packages/webapp/src/components/Animals/AddAnimalsDetails/styles.module.scss

```diff
@@ -0,0 +1,36 @@
+/*
+*  Copyright 2024 LiteFarm.org
+*  This file is part of LiteFarm.
+*
+*  LiteFarm is free software: you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation, either version 3 of the License, or
+*  (at your option) any later version.
+*
+*  LiteFarm is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+*  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+*/
+
+.section {
+  padding:16px;
+  background: var(--White);
+  border-radius: 4px;
+  border: 1px solid var(--Colors-Neutral-Neutral-100);
+}
+
+.expanded {
+  border: 1px solid var(--Form-focus);
+  box-shadow: 0px 0px 4px 0px var(--Colors-Neutral-Neutral-200);
+}
+
+.expandedContentWrapper {
+  margin-top: 16px;
+}
+
+.detailsWrapper, .sectionWrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 16px
```

Comment:
Minor, but there is a missing semi-colon here and a missing space on line 17 which I think prettier fixes automatically, so your auto-format might not be working for `.scss`.

Reply:
Huh, it still didn't add the semi-colon! So I guess it's this setting that I have checked?

![Screenshot 2024-04-30 at 11 27 03 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/052583c2-6c52-4bd1-92e1-7d1feb8930c3)

Anyway it doesn't matter for that property since it's the last, but I didn't even notice that setting was configurable before 🙂

Reply:
Sorry... 🙏 After adding the semi-colon, "CSS Language Features" seems to have removed it. Now I should properly have set it up to use prettier!
I will add the semi-colon in my next PR!